packages/storage-documents/src/on-demand-page-scan-result.ts

```diff
@@ -36,6 +37,7 @@ export interface OnDemandPageScanResult extends StorageDocument {
     priority: number;
     itemType: ItemType.onDemandPageScanRunResult;
     batchRequestId: string;
+    notificationState?: NotificationState;
```

Comment:
we should have an object that contains things like notification status, error message, error code etc

Reply:
> notification status, error message, error code etc
are not candidates for the single object, why we need that object? Example?

Reply:
pageScanResult {
run: {...},
notification: {url: string, status: enum, error: {code, message}}
}